docs/blueprints.rst

```diff
@@ -307,6 +307,84 @@ Below is an annotated example:
                                 variables["CloudFormationSpecificType"].ref))
 
 
+Utilizing Stack name within your Blueprint
+==================
```

Comment:
"The underline/overline must be at least as long as the title text."

reference: http://docutils.sourceforge.net/docs/user/rst/quickstart.html#sections